### PR TITLE
Add documentation for zmscore command

### PR DESCRIPTION
**Pull request by TysonAndre**
Docs PR for https://github.com/redis/redis/pull/7593 , which is still in review

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -3879,6 +3879,23 @@
     "since": "1.2.0",
     "group": "sorted_set"
   },
+  "ZMSCORE": {
+    "summary": "Get the score associated with the given members in a sorted set",
+    "complexity": "O(1)",
```
Comment (oranagra) [inline commands.json:3884]:
```suggestion
    "complexity": "O(N) where N is the number of fields being requested.",
```

File: commands.json [hunk 2]
```diff
@@ -3879,6 +3879,23 @@
...
+        "name": "key",
+        "type": "key"
+      },
+      {
+        "name": "member",
+        "type": "string",
+        "multiple": true
+      }
+    ],
+    "since": "1.2.0",
```
Comment (oranagra) [inline commands.json:3896]:
```suggestion
    "since": "6.2.0",
```

File: commands/zmscore.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Returns the scores associated with the specified `members` in the sorted set stored at `key`.
+
+If `key` does not exist, `nil` is returned.
```
Comment (oranagra) [inline commands/zmscore.md:3]: For every `member` that does not exist in the hash, a `nil` value is returned.

File: commands.json (hunk 1 above)
Comment (TysonAndre) [reply commands.json]: Done

File: commands.json (hunk 2 above)
Comment (TysonAndre) [reply commands.json]: Done

File: commands/zmscore.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+Returns the scores associated with the specified `members` in the sorted set stored at `key`.
+
+For every `member` that does not exist in the hash, a `nil` value is returned.
```
Comment (itamarhaber) [inline commands/zmscore.md:3]:
```suggestion
For every `member` that does not exist in the sorted set, a `nil` value is returned.
```

**Comment by oranagra** [comment]
@itamarhaber since the command is already in unstable, i think we can merge this one (whenever you're ready).

i suppose that in theory it's good to have docs visible early (client library maintainers may look at them).

**Comment by TysonAndre** [comment]
This should be ready.

**Comment by itamarhaber** [comment]
@oranagra I'm waiting for the plate to stop spinning wrt the version this will be picked from unstable - currently this PR says 6.2, but it may end up in 6.0.7 so I want to get that right.

**Pull request merged**